src/chromo/kinematics.py

```diff
@@ -76,6 +76,7 @@ class EventKinematicsBase:
     p1: Union[PDGID, Tuple[int, int]]
     p2: Union[PDGID, Tuple[int, int], CompositeTarget]
     ecm: float  # for ions this is nucleon-nucleon collision system
+    pcm: float
```

Comment:
For the record, carrying around redundant state is a maintenance burden, because it is difficult to guarantee that redundant state stays internally consistent. EventKinematics goes against this for the sake of having quick access to all needed variables without recomputation. Adding another redundant variable is therefore ok.